crates/astria-bridge-withdrawer/tests/blackbox/helpers/test_bridge_withdrawer.rs

```diff
@@ -74,25 +73,6 @@ pub(crate) const DEFAULT_IBC_DENOM: &str = "transfer/channel-0/utia";
 pub(crate) const SEQUENCER_CHAIN_ID: &str = "test-sequencer";
 const ASTRIA_ADDRESS_PREFIX: &str = "astria";
 
-static TELEMETRY: LazyLock<()> = LazyLock::new(|| {
```

Comment:
Why is this removed? The change seems to have nothing to do with this change?

Reply:
Can't pass the config into the function here.

I can strip out the new functionality of having the label off the config to roll this back.